src/DotNet.Testcontainers/Configurations/TestcontainersSettings.cs

```diff
@@ -10,6 +11,12 @@
   /// </summary>
   public static class TestcontainersSettings
   {
+    /// <summary>
+    /// Gets or sets a value indicating whether the <see cref="ResourceReaper" /> is enabled or not.
+    /// </summary>
+    public static bool ResourceReaperEnabled { get; set; }
+      = true;
```

Comment:
Should `ResourceReaperEnabled = false` also prevent adding the default `ResourceReaper` session id label to the docker resources?

Reply:
Oh, that's smart 🥇 I probably can combine this with `Initialization.StopAsync` and remove:

```csharp
if (!TestcontainersSettings.ResourceReaperEnabled)
{
    return resourceReaper;
}
```